test/plugin/test_out_google_cloud.rb

```diff
@@ -127,6 +146,18 @@ def setup
     }
   }
 
+  EC2_PARAMS = {
+    'service_name' => EC2_SERVICE_NAME,
+    'log_name' => 'test',
+    'project_id' => EC2_PROJECT_ID,
+    'zone' => EC2_PREFIXED_ZONE,
+    'labels' => {
+      "#{EC2_SERVICE_NAME}/resource_type" => 'instance',
+      "#{EC2_SERVICE_NAME}/resource_id" => EC2_VM_ID,
+      "#{EC2_SERVICE_NAME}/account_id" => EC2_ACCOUNT_ID
```

Comment:
That's an interesting question, I don't think it's ever expected that we won't have account_id set but it's possible there is some past or future version of the metadata service that does not.

I was debating whether the plugin should throw an exception in that case, but this label is not really required, so I think for now I'll leave it as-is.  I don't think there's much value in testing the "missing" case but I can add one if you feel strongly about it.